test/functional/feature_fee_estimation.py

```diff
@@ -263,6 +264,13 @@ def run_test(self):
         self.log.info("Final estimates after emptying mempools")
         check_estimates(self.nodes[1], self.fees_per_kb)
 
+        self.log.info("Testing that fee estimation is disabled in blocksonly.")
+        self.stop_nodes()
+        self.start_nodes(extra_args=[["-blocksonly"]] * len(self.nodes))
+        for node in self.nodes:
```

Comment:
You only need to test this on one node.

Reply:
Thanks, amended to only start and test on one node.